rostest: fails with "rostest not found"

When running "bitbake rostest" on a completely clean build environment, it fails in do_configure with "rostest not found".

The complete error message is:

| CMake Error at /home/bulwahn/openembedded-core/build/tmp-eglibc/sysroots/i686-linux/usr/share/catkin/cmake/find_program_required.cmake:5 (message):
|   rostest not found
| Call Stack (most recent call first):
|   build/devel/share/rostest/cmake/rostest-extras.cmake:7 (find_program_required)
|   CMakeLists.txt:22 (add_rostest)
| 
| 
| -- Configuring incomplete, errors occurred!

--- a/recipes-ros/ros-comm/rostest_1.9.41.bb
+++ b/recipes-ros/ros-comm/rostest_1.9.41.bb
@@ -3,7 +3,8 @@ SECTION = "devel"
 LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://package.xml;beginline=8;endline=8;md5=d566ef916e9dedc494f5f793a6690ba5"
 
-DEPENDS = "boost python-nose rosunit-native"
+DEPENDS = "boost python-nose rostest-native"
+DEPENDS_class-native = "boost-native rosunit-native"
 
 require ros-comm.inc
 

